### PR TITLE
Adding demo links and fixing typos

### PR DESCRIPTION
**Pull request by gadde5300**
## Description

Made changes in the following models: 

- docs/_posts/muhammetsnts/2021-08-28-bert_token_classifier_ner_jsl_en.md
- docs/_posts/murat-gunay/2021-05-03-classifierdl_bert_news_tr.md
- docs/_posts/murat-gunay/2021-07-02-nerdl_fewnerd_100d_en.md
- docs/_posts/murat-gunay/2021-09-09-classifierdl_bert_sentiment_de.md
-  docs/_posts/murat-gunay/2021-09-13-bert_token_classifier_ner_deid_en.md
- docs/_posts/murat-gunay/2021-09-16-bert_token_classifier_ner_jsl_en.md
- docs/_posts/murat-gunay/2021-09-20-bert_token_classifier_ner_drugs_en.md
- docs/_posts/murat-gunay/2021-09-28-nerdl_fewnerd_subentity_100d_pipeline_en.md

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Code improvements with no or little impact
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING](http://nlp.johnsnowlabs.com/contribute.html) page.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/_posts/muhammetsnts/2021-08-28-bert_token_classifier_ner_jsl_en.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This model is BERT-based version of `ner_jsl` model and it is better than the le
...
 
 {:.btn-box}
-<button class="button button-orange" disabled>Live Demo</button>
+[Live Demo](https://demo.johnsnowlabs.com/healthcare/NER_BERT_TOKEN_CLASSIFIER/){:.button.button-orange}{:target="_blank"}
 [Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/1.Clinical_Named_Entity_Recognition_Model.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
```
Comment (josejuanmartinez) [inline docs/_posts/muhammetsnts/2021-08-28-bert_token_classifier_ner_jsl_en.md:28]: @gadde5300 Can you please also change Open In Colab link to this one? https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/streamlit_notebooks/healthcare/NER_BERT_TOKEN_CLASSIFIER.ipynb

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Hi @gadde5300 ! 

Let's do an additional change to point the new NER BERT  notebook, aalso for bert_token_classifier_ner-jsl (en) model. Please find the comment attached. After it's fixed, please submit again the PR.

Thank you, good job!

**Comment by gadde5300** [comment]
> Hi @gadde5300 !
> 
> Let's do an additional change to point the new NER BERT notebook, aalso for bert_token_classifier_ner-jsl (en) model. Please find the comment attached. After it's fixed, please submit again the PR.
> 
> Thank you, good job!

**Review by josejuanmartinez — APPROVED** [review]
All the markdown files are updates to Modelshub documentation existing model pages at the moment of M10.1 milestone release, that needed to be adapted to newly released notebooks and live demos. Also, some typos were corrected.

**Pull request merged**